feed/history/boost_1_68_0.qbk

```diff
@@ -27,6 +27,11 @@
 
 [section Updated Libraries]
 
+* [phrase library..[@/libs/gil/ GIL]:]
+  * The I/O extensions have been entirely rewritten.
```

Comment:
@stefanseefeld I wonder, if this should also mention that the new I/O based on what was proposed by @chhenning reviewed and accepted by - Boost https://lists.boost.org/boost-announce/2011/01/0281.php I mean, to keep books in order and avoid potential questions from users if this is it or something else, etc.

Reply:
I'm not sure. 2011 is an aweful long time, especially in the context of software development. I'm not sure anyone will remember. But, this is just my opinion (which I won't defend very strongly). So if anyone wants to refine the above note, feel free !

Reply:
I just remember there have been some questions if/when it will be merged.
Following the review, there was years-long gap without significant development.

The review schedule, https://www.boost.org/community/review_schedule.html, will have to be updated anyway.